features/FxA-28-change-email/README.md

```diff
@@ -0,0 +1,53 @@
+
+Change Email
+======================
+
+https://mozilla.aha.io/features/FXA-28
+
+As an FxA user when I lose control of the email account  that is my username in Firefox Accounts I want to be able to change that FxA email address so that I don't lose access to my data and other FxA-relying services.
+* When a user changes their email in Firefox Accounts, it simply updates in other clients and relying services like display name without requiring them to reconnect.
+
+## Settings Collapsed
+* When the user visits their account settings, an **Account email** section appears under the **Display name** section.
```

Comment:
SGTM!

> Canadian English?

Yep.  It took me a surprisingly long time to figure out that I wasn't seeing it because I had my browser set to "en-GB" (but don't tell any of my fellow aussies about that...)